specs/subresourceintegrity/spec.markdown

```diff
@@ -576,7 +576,7 @@ as possible.
 <section>
 ##### HTMLLinkElement
 
-attribute DOMString integrity
+readonly attribute DOMString integrity
```

Comment:
You can invoke `setAttribute()` too so I don't think it makes much sense. For Fetch' `Request` readonly makes sense, since `Request`s are mostly immutable.